src/calendar/tests/CalendarHighlight.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react/addons';
+import CalendarHighlight from '../CalendarHighlight';
+
+const TestUtils = React.addons.TestUtils;
+
+describe('The CalendarHighlight Component', function () {
+  beforeEach(function () {
+    this.spyCx = spyOn(CalendarHighlight.prototype.__reactAutoBindMap, 'cx').and.returnValue('my-class');
```

Comment:
I wonder if we should be reaching into this level of React's abstraction layer. For instance, this method may be removed in a future version of React.